src/shared/Dropdown.tsx

```diff
@@ -0,0 +1,43 @@
+import { ChevronDown } from "lucide-solid";
+import { Component, JSX, createSignal } from "solid-js";
+import usePopper from "solid-popper";
+
+import Button from "./Button";
+
+const Dropdown: Component<{ children: JSX.Element; label: string }> = (
+  props
+) => {
+  const [open, setOpen] = createSignal(false);
+  const [dropdown, setDropdown] = createSignal();
+  const [dropdownMenu, setDropdownMenu] = createSignal();
+
+  usePopper(dropdown, dropdownMenu, {
+    placement: "bottom-start",
+  });
```

Comment:
Awesome, thanks! One last feature request: is it possible to configure popper so that clicking outside of the dropdown automatically closes it? Just now realized that this isn't the default behavior.

Reply:
added in latest commit